pkg/cli/zip_test.go

```diff
@@ -838,3 +847,114 @@ func TestNodeRangeSelection(t *testing.T) {
 		}
 	}
 }
+
+func TestZipJobTrace(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+	defer jobs.ResetConstructors()()
+
+	ctx := context.Background()
+	s, sqlDB, _ := serverutils.StartServer(t, base.TestServerArgs{
+		Insecure:          true,
+		DefaultTestTenant: base.TestControlsTenantsExplicitly,
```

Comment:
Maybe a comment that we want to use an explicit system tenant because the format of the debug.zip differs depending on the tenant that the job lives in.

Reply:
i actually don't know how this ended up here. This test seems to work fine for tenants as well, does this surprise you?

Reply:
Now that I read the test more closesly, it doesn't surprise me.